bigquery/tables.js

```diff
@@ -177,36 +128,53 @@ function importFile (options, callback) {
       .on('error', callback)
       .on('complete', function (metadata) {
         console.log('Completed job: %s', job.id);
-        return callback(null, metadata);
+        return callback(null, metadata, apiResponse);
+      });
+  });
+}
+
+// [START import_file_from_gcs]
+var Storage = require('@google-cloud/storage');
+
+function importFileFromGCS (datasetId, tableId, bucketName, fileName, callback) {
+  var bigquery = BigQuery();
+  var storage = Storage();
+
+  var table = bigquery.dataset(datasetId).table(tableId);
+  var file = storage.bucket(bucketName).file(fileName);
+
+  // Import the file from Google Cloud Storage
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/bigquery/latest/bigquery/table?method=import
+  table.import(file, function (err, job, apiResponse) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Started job: %s', job.id);
+    job
+      .on('error', callback)
+      .on('complete', function (metadata) {
+        console.log('Completed job: %s', job.id);
+        return callback(null, metadata, apiResponse);
       });
   });
 }
-// [END import_file]
+// [END import_file_from_gcs]
 
+/* eslint-disable no-redeclare */
 // [START export_table_to_gcs]
-/**
- * Export a table from BigQuery to Google Cloud Storage.
- *
- * @param {object} options Configuration options.
- * @param {string} options.bucket A Google Cloud Storage bucket to use for storage.
- * @param {string} options.file The file to save results to within Google Cloud Storage.
- * @param {string} options.dataset The ID of the dataset to use.
- * @param {string} options.table The ID of the project to use.
- * @param {string} options.format Format to export as - either 'CSV', 'JSON', or 'AVRO'.
- * @param {boolean} [options.gzip] Optional. Whether or not data should be compressed using GZIP.
- * @param {function} callback Callback function to receive query results.
- */
-function exportTableToGCS (options, callback) {
-  var gcsFileObj = storage.bucket(options.bucket).file(options.file);
-  var table = bigquery.dataset(options.dataset).table(options.table);
-  var config = {
-    format: options.format,
-    gzip: options.gzip
-  };
+var Storage = require('@google-cloud/storage');
```

Comment:
Yes, because this is an import. Imports should be as close to top-level statements as possible. In ES6, the new version of JavaScript, this is enforced by the new `import` syntax, which only works as a top-level statement.